scene/gui/scroll_container.cpp

```diff
@@ -434,24 +434,8 @@ void ScrollContainer::update_scrollbars() {
 
 	Size2 min = child_max_size;
 
-	bool hide_scroll_v = !scroll_v || min.height <= size.height;
```

Comment:
I just moved these lines so that'd it'd be horizontal first, then vertical, for consistency sake